### PR TITLE
2020 04

### PR DESCRIPTION
**Pull request by czubair**

### CODE REVIEW

File: releases/2020-04/dotnet.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+The Azure SDK team is pleased to announce our April 2020 client library releases.
+
+#### GA
+
+- Event Hubs
+- Key Vault (Certificates)
+
+#### Updates
+
+- Key Vault (Keys, Secrets)
```
Comment (heaths) [inline releases/2020-04/dotnet.md]: I know we're not releasing anything, so I'm guessing this is carry-over. I recommend clearing it out for new releases so misinformation doesn't leak in accidentally.

Comment (czubair) [reply releases/2020-04/dotnet.md]: Okay, will do that, Heath!

File: releases/2020-04/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Azure SDK April 2020
+layout: post
+date: April 2020
+tags: release
+sidebar: releases_sidebar
+permalink: /releases/2020-02/index.html
```
Comment (alzimmermsft) [inline releases/2020-04/index.md]: Should this be `/releases/2020-04/index.html`? Given this was copied from 2020-03 is the link incorrect there as well?

Comment (czubair) [reply releases/2020-04/index.md]: Yes, I will change that. Thanks, Alan!

**Pull request merged**